cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/config/ConfigurationFinderTest.java

```diff
@@ -74,6 +78,7 @@ public void invalidInputConfigurationTest() {
    *
    */
   @Test
+  @Ignore
```

Comment:
Would be nice if there is a comment/hint on why these tests were ignored and on which ticket these should be activated